docs/source/user-guide/general/ci_cd.rst

```diff
@@ -2,60 +2,62 @@
+can be incorporated into the process, including code snippets that can be used for running deepchecks
+with Airflow or Github Actions.
 
 **Structure:**
 
-* `CI/CD In Machine Learning <#ci/cd-in-machine-learning>`__
+* `Intro to CI/CD In Machine Learning <#ci/cd-in-machine-learning>`__
 * `Airflow Integration <#airflow-integration>`__
 * `GitHub Actions Integration <#github-actions-integration>`__
 
 CI/CD In Machine Learning
 ==========================
 
 CI/CD is a software engineering concept that is used to streamline the process of building, testing and deploying
-software products. CI/CD can also be utilized for the ML model lifecycle - to streamline the process of model training
-(and retraining), model validation and model deployment.
-
-For example, CI/CD in machine learning can be used in different steps such as:
-
-* Data integrity validation: When the data used for training is collected via automatic processes and pipelines,
-  the data may contain errors and problems we haven't encountered before, either due to a bug in the
-  data processing pipeline or due to a change in the data source.
-  Examples of such problems include:
-  :doc:`conflicting labels between similar samples</checks_gallery/tabular/data_integrity/plot_conflicting_labels>`,
-  :doc:`high correlation between features</checks_gallery/tabular/data_integrity/plot_feature_feature_correlation>`,
-  :doc:`spelling errors in categorical features</checks_gallery/tabular/data_integrity/plot_string_mismatch>`,
-  and more.
-* Datasets comparison: In many cases it's useful to make sure that there isn't any leakage or drift between 2
-  datasets. For example, when doing a time based split of the data there is a risk that the datasets will have
-  significant differences, or when doing a periodic model retraining we might want to compare the new dataset
-  to the previous one.
-  Examples of checks that can be used are:
-  :doc:`drift between features</checks_gallery/tabular/train_test_validation/plot_train_test_feature_drift>`,
-  :doc:`change in correlation between features and label</checks_gallery/tabular/train_test_validation/plot_feature_label_correlation_change>`,
-  :doc:`duplicate samples between the datasets</checks_gallery/tabular/train_test_validation/plot_train_test_samples_mix>`,
-  and more.
-* Model training: The model is trained on the (previously validated) training set.
-* Model validation: The trained model is evaluated on test data, testing for performance, weak segments and more:
+software products. These can be made more trustworthy and efficient, and help improve the R&D processes,
+by adding automated steps to the development and deployment lifecycle.
+For ML models, CI/CD concepts can be utilized to streamline the process of model training
+(and retraining), data and model validation and model deployment:
+
+* | **Data integrity validation**: When the data used for training is collected via automatic processes and pipelines,
```

Comment:
What does this do?

Reply:
it enables a new line which is still under the same bullet